homeworks/editerdelyhorvath/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,53 @@
+'''
+2. házi feladat
+
+Feladat 1: Változók, user input, string metódusok, type conversion, 
+f-string használata
+
++ Extra feladat (szorgalmi)
+'''
+import os
+
+# Function to clear the terminal screen
+def clear_terminal():
+    if os.name == 'nt':  # Windows
+        os.system('cls')
+    else:  # Mac and Linux
+        os.system('clear')
+
+clear_terminal()
+
+# Ask the user for input
+name = input("Enter your name: ").strip().upper()
+age_in_years = int(input("Enter your age (in years): ").strip())
+python_exp_in_years = float(input("Enter your Python experience (in years): ").strip())  # allowing a fraction of a year - eg half a year
+
+# EXTRA - Ask if the character wants to be a professional Python developer
+wants_to_be_dev_input = input("Do you want to be a professional Python developer? (yes/no): ").strip().lower()
+
+# Using ternary operator to store the boolean value
+wants_to_be_dev = True if wants_to_be_dev_input == "yes" else False
+
+# Convert age to days (assuming 365 days per year)
+age_in_days = age_in_years * 365
+
+clear_terminal()
+
+# Print the result
+print(f"Your name is {name}, you are {age_in_days:.0f} days old and you has {python_exp_in_years} years of Python experience.")
+print(f"{'I am happy you want' if wants_to_be_dev else 'I am sad that you do not want'} to be a Python developer.\n\n")
```

Comment:
Mi lenne ha a 29. soron a wants_to_be_dev már eleve ezt a 2 stringet tudná felvenni amit itt megírsz?
'I am happy you want'
'I am sad that you do not want'
így nem kéne még egy ternary kifejezés a printbe.
A másik dolog: ki tudsz ugyanígy printelni több sort is egy print()-ből ha multi line f-stringet csinálsz, és ha szükséges használod a newline escape character-t (\n)
```
print("""

""")
```

Reply:
Így tényleg menőbb, köszönöm a javaslatot!